### DISCUSSION TITLE
Support for SyncedStore

### DISCUSSION THREAD

**Discussion by andrezimpel** [ueberdosis/hocuspocus#497] [Q&A] [answered]
I try to wrap my head around how I can use Hocuspocus with SyncedStore. Setting up HocuspocusProvider does work to sync data between browsers and offline support works perfectly too, but I cannot get the data on the server in order to store it. 

I might be missing something so any help would be appreciated a lot! Thank you!

**Comment by mkriegeskorte** [comment, marked as answer]
Hi @andrezimpel,

did you set up the Hocuspocus server, too? With the database extension you should be able to persist the database in a storage of your choice. https://tiptap.dev/hocuspocus/api/extensions/database

There shouldn’t be any need to use SyncedStore :)
